### ISSUE TITLE
async on_ws_frame hooks

### ISSUE THREAD

**Issue by leftys** [tarasko/picows#6]
Hi, do you have any idea how to make reading the data asynchronous? Eg. making the on_ws_frame function async? I tried `asyncio.create_task` and pushing the data into `asyncio.Queue`, but both is pretty slow.

**Comment by tarasko** [comment]
Yes, that was the reason why I decided not to provide async iteration interface like aiohttp and websockets do. 
Because both `create_task` and Queue are slow and introduce significant delay and a lot of churn for the event loop.

Which is very undesirable when there is a spike in market activity.
